### ISSUE TITLE
Mute linking not working on restart

### ISSUE THREAD

**Issue by kineticscreen** [exeldro/obs-audio-monitor#33]
I have a monitor audio filter on the Mic / Aux 1 input, and have it set that the mute status on the Monitor is linked to the source mute. However on starting up OBS this doesn't work, and I need to set the link to 'not linked' and then back to 'linked' in order for it to operate correctly.

**Comment by Trouffman** [comment]
Could you share more details on your system & how to reproduce this bug ?

Please help by providing : 
OBS version (in Help > About menu)
Plugin version used (can see in the log file)

Close OBS, do a clean start and re-launch, do the step you are describing and then Help > Log Files > Upload last log files. and procide the link here.

**Comment by kineticscreen** [comment]
Yup that appears to be the same issue - was suing Mic/Aux input.
